src/constructs/dynamodb/dynamodb.ts

```diff
@@ -0,0 +1,41 @@
+import { Tags } from "aws-cdk-lib";
+import { Table } from "aws-cdk-lib/aws-dynamodb";
+import type { TableProps } from "aws-cdk-lib/aws-dynamodb";
+import type { GuStack } from "../core";
+
+export interface OptIn {
+  enabled: true;
+}
+
+export interface OptOut {
+  enabled: false;
+  /**
+   * We recommend using DevX Backups where possible. If it is not suitable for your use-case please document
+   * this here so that we can understand why this is switched off when performing security audits.
+   */
+  optOutReason: string;
+}
+
+export interface GuDynamoTableProps extends TableProps {
+  /**
+   * We recommend using DevX Backups to protect your DynamoDB table's backups.
+   * For more details on this feature, see the
+   * [documentation](https://docs.google.com/document/d/11EZtuVHCnjavE9AYLroiuDUsMzbp2ulFBtj4VkyCKNc/edit?usp=sharing).
+   */
+  devXBackups: OptIn | OptOut;
+}
+
+/*
+ * D
+ */
+export class GuDynamoTable extends Table {
+  /**
```

Comment:
I think this documentation/comment would typically live above the class rather than above the constructor. There is also a comment fragment above so perhaps this was moved accidentally?

Reply:
Not sure where that partial comment was from - deleted it anyway and moved documentation/comment above class declaration.